### PR TITLE
fix send rr msg bug

### PR DESCRIPTION
**Pull request by lrhkobe**
fix send rr msg bug #188 
- fix rr reply bug
- rollback delete config file in eventmesh-common
- remove setInstanceName method in eventmesh-connector-api

### CODE REVIEW

File: eventmesh-common/src/main/java/com/webank/eventmesh/common/config/ConfigurationWraper.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class ConfigurationWraper {
```
Comment (wqliang) [inline eventmesh-common/src/main/java/com/webank/eventmesh/common/config/ConfigurationWraper.java:33]: here is a spell mistake ConfigurationWrapper , you can fix it later

**Review by wqliang — COMMENTED** [review]
this looks good to me

File: eventmesh-connector-rocketmq/src/main/java/com/webank/eventmesh/connector/rocketmq/consumer/PushConsumerImpl.java [hunk 2]
```diff
@@ -83,7 +83,8 @@ public PushConsumerImpl(final KeyValue properties) {
         this.rocketmqPushConsumer.setMessageModel(MessageModel.valueOf(clientConfig.getMessageModel()));
 
         String consumerId = OMSUtil.buildInstanceName();
-        this.rocketmqPushConsumer.setInstanceName(consumerId);
+        //this.rocketmqPushConsumer.setInstanceName(consumerId);
```
Comment (xwm1992) [inline eventmesh-connector-rocketmq/src/main/java/com/webank/eventmesh/connector/rocketmq/consumer/PushConsumerImpl.java:86]: you can remove this unused code

File: eventmesh-runtime/src/main/java/com/webank/eventmesh/runtime/core/protocol/http/consumer/ProxyConsumer.java [hunk 3]
```diff
@@ -67,20 +67,20 @@ public synchronized void init() throws Exception {
         keyValue.put("isBroadcast", "false");
         keyValue.put("consumerGroup", consumerGroupConf.getConsumerGroup());
         keyValue.put("proxyIDC", proxyHTTPServer.getProxyConfiguration().proxyIDC);
+        keyValue.put("instanceName", ProxyUtil.buildProxyClientID(consumerGroupConf.getConsumerGroup(),
+                proxyHTTPServer.getProxyConfiguration().proxyRegion,
+                proxyHTTPServer.getProxyConfiguration().proxyCluster));
         persistentMqConsumer.init(keyValue);
 
         //
```
Comment (xwm1992) [inline eventmesh-runtime/src/main/java/com/webank/eventmesh/runtime/core/protocol/http/consumer/ProxyConsumer.java:75]: this is unnecessary code

**Comment by xwm1992** [comment]
this pull request is ok, some unnecessary code can remove later

**Pull request merged**